### PR TITLE
Mhv 60769 updates for the mr landing page

### PR DESCRIPTION
**Pull request by Wyzardsleeves**
## Summary

- Landing page with content from [the design](https://www.figma.com/design/mGZRdLypKGaFsHo5xp2kaZ/Medical-Records?node-id=6717-44444&t=LevUBhYIXPOK8ryg-4).
- This is not a bug.
- Updated the JSX. 
- MHV Medical Records.

### [MHV-60769](https://jira.devops.va.gov/browse/MHV-60769) Updates for the MR landing page-needed before launch to final Phase 0 ###
Screenshot 2024-08-16 at 2 11 02 PM

## Screenshot
Screenshot 2024-08-19 at 9 57 58 AM

MHV-60768 Screenshot
Screenshot 2024-08-20 at 11 00 25 AM

## Testing done

-  Updated Unit test for JSX.

## What areas of the site does it impact?
MHV Medical Records

### CODE REVIEW

File: src/applications/mhv-medical-records/containers/LandingPage.jsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ const LandingPage = () => {
   const displayConditions = useSelector(selectConditionsFlag);
   const displayVitals = useSelector(selectVitalsFlag);
   const displayLabsAndTest = useSelector(selectLabsAndTestsFlag);
+  const displayMedicalRecordsSettings = true; // until "useSelector(selectSettingsPageFlag)" is working;
```
Comment (mmoyer-va) [inline src/applications/mhv-medical-records/containers/LandingPage.jsx:37]: I suspect this should be `false` until the flag is working.

**Review by mmoyer-va — CHANGES_REQUESTED** [review]
Your screenshot doesn't appear related to the ticket.

File: src/applications/mhv-medical-records/containers/App.jsx [hunk 2]
```diff
@@ -290,7 +290,7 @@ const App = ({ children }) => {
                 )}
                 <div className="vads-l-grid-container vads-u-padding-x--0 vads-u-margin-x--0 vads-u-flex--fill">
                   <div className="vads-l-row">
-                    <div className="vads-l-col">{children}</div>
+                    <div className="vads-l-col--8">{children}</div>
```
Comment (mmoyer-va) [inline src/applications/mhv-medical-records/containers/App.jsx:293]: This `--8` is causing issues with responsive layout. What was the reason behind this change?

image

File: src/applications/mhv-medical-records/containers/LandingPage.jsx [hunk 3]
```diff
@@ -191,6 +193,27 @@ const LandingPage = () => {
...
+            settings.
+          </p>
+          <Link
+            to={mhvUrl(
+              isAuthenticatedWithSSOe(fullState),
+              'electronic-record-sharing-options',
+            )}
+            className="vads-c-action-link--blue"
+            data-testid="settings-link"
+          >
```
Comment (mmoyer-va) [inline src/applications/mhv-medical-records/containers/LandingPage.jsx:212]: Just verifying that this link is supposed to point to MHV Classic, is that right?

**Review by mmoyer-va — CHANGES_REQUESTED** [review]

File: src/applications/mhv-medical-records/containers/LandingPage.jsx (hunk 3 above)
Comment (Wyzardsleeves) [reply src/applications/mhv-medical-records/containers/LandingPage.jsx:212]: yes I got it from UCD. I imitated a similar link and changed the endpoint.

File: src/applications/mhv-medical-records/containers/App.jsx (hunk 2 above)
Comment (Wyzardsleeves) [reply src/applications/mhv-medical-records/containers/App.jsx:293]: I saw instructions that said to make the column "8 columns wide" and I thought that's what they meant. I can't find i on the Figma anymore so I guess somebody changed their mind imma change it back.

**Review by mmoyer-va — APPROVED** [review]
Thanks for making the fix -- looks good!

**Pull request merged**